src/app.js

```diff
@@ -130,8 +130,6 @@ app.use(
 // When a text/html request is received, negotiate and store the requested language
 // Using asyncLocalStorage avoids having to pass req context down through every function (e.g. getMessage())
 app.use((req, res, next) => {
-  if (!req.headers.accept?.startsWith('text/html')) return next()
```

Comment:
This is the actual fix of this PR. The changes in the other file don't change the behaviour, but fix a potential issue where we relied on implicit casting, described in more detail in 4ee5ec74.